### ISSUE TITLE
Why is IoU not mentioned during training?

### ISSUE THREAD

**Issue by working12** [HuguesTHOMAS/KPConv-PyTorch#174]
The code provided does not provide us with the IoU for the training case. It only prints the loss. How can we add that? What is the easiest way to include that?

**Comment by HuguesTHOMAS** [comment]
IoU on training set does not make much sense, but if you want to have it, you can for example, add the training cloud to the list of validation clouds.

IF you really want to print IoU on the training clouds directly, it would require significant changes to the code
